src/lava/utils/float2fixed.py

```diff
@@ -1,26 +1,719 @@
+               the passed model without hierarchical Processes. In the
+               following, the conversion will be performed on the new model.
+               Redo step 2 with the new model and continue with 4.
+            4. Finding and instantiating the fixed-point ProcessModels.
+            5. Retrieving the information needed for conversion from the
+               fixed-point ProcessModels chosen via the provided fixed-point
+               run configuration.
+            5. Creating monitors for recording the activation statistics of the
+            6  dynamical variables of Processes. With this, the domain of the
+               variable can be determined to constrain the floating- to
+               fixed-point conversion.
+            7. Execution of the floating-point model chosen via run
+               configuration. If non-default quantile information is passed the
+               domain of the dynamical variables will be smaller than taking
+               just the minimum and maximum of the recorded data.
+            8. Updating the scale_domains, i.e. the collection of variables
+               that need to be scaled identically.
+            9. Finally, scale the parameters so that no over/underflow occurs
+               given the variability of the variables recorded in step 4.
+
+        Parameters
+        ----------
+
+        proc : AbstracProcess or List[AbstractProcess]
+            Processes to be converted
+        num_steps : int
+            Number of steps the floating-point model is executed for. A larger
+            number of steps will lead to better statistics of the monitored
+            variables which is needed for the conversion but also will lead to
+            a longer run time
+        quantiles : list
+            List of two numbers both in the interval [0,1] determining the
+            p-quantile of the domain of a monitored variable to set largest and
+            smallest value of the dynamic rep_range of that variable. The
+            dynamic range will be used to determine the optimal floating- to
+            fixed-point conversion. The first entry must be smaller than the
+            second. 0 resp. 1 correspond to the minimum resp. maximum of the
+            domain.
+        find_connected_procs : bool
+            Find all Processes connected to passed Process
+        """
+        self.num_steps = num_steps
+        self.quantiles = quantiles
+        self._set_procs(proc=proc, find_connected_procs=find_connected_procs)
+        # Infer Variables and Ports of Processes.
+        self.var_ports = self._get_var_ports()
+        # If hierarchical Processes are involved, set up copy of model where
+        # Subprocesses are made explicit.
+        if self.hierarchical_procs:
+            self._explicate_hierarchical_procs()
+        else:
+            pass
+        # Get fixed-point ProcessModels of Processes.
+        self.fixed_pt_proc_models = self._get_fixed_pt_proc_models()
```

Comment:
What's the difference between line 124 and 126?

Reply:
Nice, should be addressed in 58fd18a6c5ec776b81b991a6260f4edcc2d2ca2d.